scripts/Calibration/tube.py

```diff
@@ -420,7 +420,7 @@ def calibrate(ws, tubeSet, knownPositions, funcForm, **kwargs):
         margin = param_helper.get_parameter(MARGIN, kwargs)
         fit_par.setMargin(margin)
 
-    range_list = param_helper.get_parameter(RANGELIST, kwargs, default_range_list=tubeSet.getNumTubes())
```

Comment:
The tube calibration now works, but the script produces another error concerning workspace pointer when applying the calibration, which I take to be another issue.